code/modules/food_and_drinks/machinery/processor.dm

```diff
@@ -238,8 +238,8 @@
 		processed_slime.forceMove(drop_location())
 		processed_slime.balloon_alert_to_viewers("crawls free")
 		return
-	var/core_count = processed_slime.cores
-	for(var/i in 1 to (core_count+rating_amount-1))
+
+	for(var/i in 1 to rating_amount)
```

Comment:
You didn't fully revert the nerf smh.
```suggestion
	for(var/i in 1 to (processed_slime.cores+rating_amount-1))
```

Reply:
What nerf? 

The processing always gives 1 to 4 depending on the parts tier, regardless of the slime size.
The surgery gives 1 core from baby slime and 2 cores from adult slime.

Reply:
You aren't taking into account the slime steroid potion (which I'm not sure it's even useful) which increases the amount of cores a slime has.